Acornfile

```diff
@@ -20,84 +20,92 @@ services: db: {
 	data: dbName: args.dbName
 }
 
-containers: {
-  mongodb: {
-    name: "MongoDB"
-    description: "Container running a MongoDB server"
-    image: "mongo:6.0"
-    ports: "27017/tcp"
-    env: {
-      MONGO_INITDB_ROOT_USERNAME: "secret://admin/username"
-      MONGO_INITDB_ROOT_PASSWORD: "secret://admin/password"
-    }
-    dirs: {
-      "/data/db": "volume://db-data"
-    }
-    probes: [
-      {
-        type:                "liveness"
-        initialDelaySeconds: 10
-        timeoutSeconds:      5
-        tcp:{
-          url: "tcp://localhost:27017"
-        }
-      },   
-    ]
-  }
+containers: mongodb: {
+	name:        "MongoDB"
+	description: "Container running a MongoDB server"
+	image:       "mongo:6.0"
+	ports:       "27017/tcp"
+	env: {
+		MONGO_INITDB_ROOT_USERNAME: "secret://admin/username"
+		MONGO_INITDB_ROOT_PASSWORD: "secret://admin/password"
+		"REPLICA_SET_KEY":          "secret://replica-set-key/token"
```

Comment:
It would be nice to mount the content of a generated secret as a file, but mongod requires that we run `chmod 999:999` on the file before it will read it.